api/orchestrator/orchestrator.proto

```diff
@@ -1079,6 +942,14 @@ message ListControlsRequest {
   string catalog_id = 1;
   string category_name = 2;
 
+  message Filter {
+    // Optional. Lists only controls with the specified assurance levels.
+    repeated string assurance_levels = 1
+        [ (validate.rules).repeated .items.string.min_len = 1 ];
```

Comment:
```suggestion
        [ (validate.rules).repeated.items.string.min_len = 1 ];
```

Reply:
My auto-formatting always does this :( The problem is that this will return once I save the file again.

Reply:
OK, then I will ignore it in future.